src/DynamoManipulation/TranslationGizmo.cs

```diff
@@ -170,25 +169,25 @@ public void UpdateGeometry(Point origin, Vector axis1, Vector axis2, Vector axis
             planes.Clear();
             
             scale = size;
-            position = origin;
+            Origin = origin;
             var col = Convert.ToByte(255);
-            
-            axes.Add(new AxisDescriptor() { Axis = axis1, Color = Color.FromRgb(col, 0, 0), Name = "_xAxis" });
+
+            axes.Add(axis1);
             if (axis2 != null)
             {
-                axes.Add(new AxisDescriptor() { Axis = axis2, Color = Color.FromRgb(0, col, 0), Name = "_yAxis" });
-                planes.Add(Plane.ByOriginXAxisYAxis(position, axis1, axis2));
+                axes.Add(axis2);
+                planes.Add(Plane.ByOriginXAxisYAxis(Origin, axis1, axis2));
             }
             if (axis3 != null)
             {
-                axes.Add(new AxisDescriptor() { Axis = axis3, Color = Color.FromRgb(0, 0, col), Name = "_zAxis" });
+                axes.Add(axis3);
                 if (axis2 != null)
-                    planes.Add(Plane.ByOriginXAxisYAxis(position, axis2, axis3));
-                planes.Add(Plane.ByOriginXAxisYAxis(position, axis3, axis1));
+                    planes.Add(Plane.ByOriginXAxisYAxis(Origin, axis2, axis3));
```

Comment:
Thanks for reminding me. I prefer to write compact code, hence always get back to this mode. I'll try to follow this.